src/plugins/data/server/index.ts

```diff
@@ -312,3 +312,5 @@ export const config: PluginConfigDescriptor<ConfigSchema> = {
 };
 
 export type { IndexPatternsService } from './index_patterns';
+
+export { correlationsSavedObjectType } from './saved_objects';
```

Comment:
were we intending to export the type correlations . not sure if the saved object type needed to be exported